ddl/db_test.go

```diff
@@ -2163,13 +2163,17 @@ func newTestMaxTableRowIDContext(c *C, d ddl.DDL, tbl table.Table) *testMaxTable
 	}
 }
 
+type tableMaxRowID interface {
```

Comment:
Why not use `DDL` interface directly?

Reply:
But the `ddlCtx` implements GetTableMaxRowID() interface.